src/main/java/Reacher.java

```diff
@@ -0,0 +1,91 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Reacher {
+    public static void main(String[] args) {
+            Scanner scanner = new Scanner(System.in);
+        ArrayList<Task> memory = new ArrayList<>();
+        System.out.println("Hello!\n" +
+                "I'm Reacher.\n" +
+                "Give me tasks.\n" +
+                "Functions are edit, list, delete and bye");
+        while (true) {
+            try {
+                String input = scanner.nextLine();
+                if (input.isEmpty()) {
+                    throw new ReacherException("pls type a task name.");
+                }
+                if (input.equalsIgnoreCase("bye")) { //check for end request
+                    System.out.println("bye");
+                    break;
+                } else if (input.equalsIgnoreCase("list")) {// check for list request
+                    System.out.println("Tasks:");
+                    int c = 1;
+                    for (Task task : memory) {
```

Comment:
Perhaps a normal for loop would be better here? Then the declaration of c would be within the same scope as the for loop.